tests/models.py

```diff
@@ -11,7 +10,7 @@ class CustomField(models.CharField):
 
     def __init__(self, *args, **kwargs):
         kwargs['max_length'] = 12
-        super(CustomField, self).__init__(*args, **kwargs)
+        super().__init__(*args, **kwargs)
```

Comment:
That part can stay. It helps typing tools.
```suggestion
        super(CustomField, self).__init__(*args, **kwargs)
```